tests/table/test_init.py

```diff
@@ -689,7 +689,7 @@ def test_update_metadata_add_snapshot(table_v2: Table) -> None:
         snapshot_id=25,
         parent_snapshot_id=19,
         sequence_number=200,
-        timestamp_ms=1602638573590,
+        timestamp_ms=1602638593590,
```

Comment:
Seems like an un-related change?

Reply:
This change is relevant now, I missed to update his comment.
Basically I'm keeping the timestamp of snapshot to be higher than last-updated-ms of the table.

Reply:
Ah I see. The previous value collides with the example table's `last_udpated_ms` and thus make update_table_metadata thinks we need a new timestamp. Thanks for the explanation!